CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 
 ### Bug fixes
 
+* [#7391](https://github.com/rubocop-hq/rubocop/issues/7391): Support pacman formatter on Windows. ([@laurenball][])
```

Comment:
You'll need to move this under `master`, as we just cut 0.75.

Reply:
Thank you! I'm planning to look at this some more this evening :)